README.md

```diff
@@ -80,10 +80,38 @@ jobs:
           app-id: ${{ vars.APP_ID }}
           private-key: ${{ secrets.PRIVATE_KEY }}
       - id: committer
-        run: echo "string=${{steps.app-auth.outputs.app-slug}}[bot] <${{ steps.app-auth.outputs.installation-id }}+${{ steps.app-auth.outputs.app-slug }}[bot]@users.noreply.github.com>"  >> "$GITHUB_OUTPUT"
+        run: echo "string=${{steps.app-token.outputs.app-slug}}[bot] <${{ steps.app-token.outputs.installation-id }}+${{ steps.app-token.outputs.app-slug }}[bot]@users.noreply.github.com>"  >> "$GITHUB_OUTPUT"
       - run: echo "committer string is ${{steps.committer.outputs.string}}"
 ```
 
+### Configure git CLI for an app's bot user
+
+```yaml
+on: [pull_request]
+
+jobs:
+  auto-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          # required
+          app-id: ${{ vars.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
+      - run: |
+          git config --global user.name '${{steps.app-token.outputs.app-slug}}[bot]'
+          git config --global user.email '<BOT USER ID>+${{ steps.app-token.outputs.app-slug }}[bot]@users.noreply.github.com>'
+      # git commands like commit work using the bot user
+      - run: |
+          git add .
+          git commit -m "Auto-generated changes"
+          git push
+```
+
+The `<BOT USER ID>` is the numeric user ID of the app's bot user, which can be found under `https://api.github.com/users/<app-slug>%5Bbot%5D`.
```

Comment:
Thanks @gr2m - when using this example in practice, I noticed that it doesn't seem correct to use the installation ID in the committer email but it should be the user ID of the bot. If I followed the above committer string example, then I get a commit in the UI that displays two different users

https://github.com/wasilibs/go-protoc-gen-builtins/commit/9c7fdf733f02b5fbaed8a4fb79960e7d397bd01d

I looked at `git log` for that commit and noticed that it was merged using the bot's user ID, so I changed to use it for the email, and then it shows as expected, a single committer

https://github.com/wasilibs/go-protoc-gen-builtins/commit/95931e46d68e0523e0e1c7fb7c58112f865ba732

So I suspect this form is correct. I'm not sure exactly how the committer string example is meant to be used, but I guess it's a similar flow so should I update it to user bot user ID like here? Should the action add to `outputs`, `bot-user-id` so users don't have to look it up?

Thanks.